clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/GvcfTests.scala

```diff
@@ -770,47 +766,48 @@ trait GvcfTests { self: BaseIntegrationSpec =>
       result <- query
     } yield {
       result.regulatoryDesignation should be(
-        Some(RegulatoryDesignation.ResearchOnly)
+        Some(RegulatoryDesignation.ClinicalDiagnostics)
       )
     }
   }
 
-  it should "respect user-set regulatory designation for gvcfs" in {
-    val project = s"project$randomId"
-    val sample = s"sample$randomId"
-    val version = 3
-
-    val cloudPath = rootTestStorageDir.resolve(
-      s"gvcf/$project/$sample/v$version/$randomId.gvcf"
+  it should "preserve any existing regulatory designation for gvcfs" in {
```

Comment:
We don't have a wrapper for gvcfs that does partial upserts (yet), but this covers the same scenario as the cram delivery process.